sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -573,6 +575,21 @@ final class DataFrameWriter[T] private[sql](ds: Dataset[T]) {
     format("csv").save(path)
   }
 
+  private def runCommand(session: SparkSession, name: String)(command: LogicalPlan): Unit = {
```

Comment:
Add a function description like?

  /**
   * Wrap a DataFrameWriter action to track the QueryExecution and time cost, then report to the
   * user-registered callback functions.
   */